tests/training/metrics/auc_test.py

```diff
@@ -100,6 +100,7 @@ def test_distributed_auc(self):
         labels = torch.randint(3, 5, (8,), dtype=torch.long)
         # We make sure that the positive label is always present.
         labels[0] = 4
+        labels[4] = 4
```

Comment:
This is unrelated?

Reply:
Unrelated. Might need this on master too actually.